### PR TITLE
Update sourabh_bajaj_resume.tex

### PR DESCRIPTION
**Pull request by NunoSempere**
Would not work without the babel package.

### CODE REVIEW

**Comment by NunoSempere** [comment]
Added "\usepackage[english]{babel}", without which the document wouldn't compile

File: sourabh_bajaj_resume.tex [hunk 1]
```diff
@@ -15,6 +15,8 @@
 \usepackage{enumitem}
 \usepackage[hidelinks]{hyperref}
 \usepackage{fancyhdr}
+\usepackage[english]{babel}
+
```
Comment (sb2nov) [inline sourabh_bajaj_resume.tex]: remove blank line?

Comment (NunoSempere) [reply sourabh_bajaj_resume.tex]: Yeah, the blank line does nothing.

**Comment by sb2nov** [comment]
Thanks 👍

**Pull request merged**